### PR TITLE
Param change & single eval test

### PR DESCRIPTION
**Pull request by sbpodonos**
### 🚀 Pull Request Checklist

- [x] Summarize the changes made in this pull request.
- [x] Tested the changes to ensure they work as expected.
- [x] Updated relevant documentation for the code changes.

### 📋 Description

Some comment update on params

### CODE REVIEW

File: tests/single_eval_test.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    python single_eval_test.py --api_key=<MY_API_KEY> --base_url=<BASE_URL>
+"""
+
+import argparse
+import podonos
+from podonos import *
+from podonos.core.base import *
+import sys
+
+_PODONOS_API_BASE_URL = "https://prod.podonosapi.com"
```
Comment (PodonosKim) [inline tests/single_eval_test.py:14]: I think this is no need to be exposed

**Review by PodonosKim — APPROVED** [review]
LGTM

File: tests/single_eval_test.py (hunk 1 above)
Comment (sbpodonos) [reply tests/single_eval_test.py]: you are right.

**Review by PodonosKim — APPROVED** [review]
LGTM

**Pull request merged**